src/Installers/Windows/SharedFrameworkBundle/1029/thm.wxl

```diff
@@ -1,14 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="en-us" Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
-  <String Id="Caption">Instalační program pro [BundleNameFull]</String>
+  <String Id="Caption">[BundleNameFull] Setup</String>
```

Comment:
Why do some languages get reverted to English?

Reply:
Translations haven't flowed for those languages yet, so they're being reverted.